File: solarized.py
"""Solarized color terminal.

This module contains some mappings for the ANSI terminal escapes support
the 16 colors of the Solarized color palette created by Ethan Schoonover.
"""

base03 = b03 = Base03 = B03 = '\x1b[1;30m'
base03 = b03 = Base03 = B03 = '\x1b[1;30m'
base02 = b02 = Base02 = B02 = '\x1b[0;30m'
base01 = b01 = Base01 = B01 = '\x1b[1;32m'
base00 = b00 = Base00 = B00 = '\x1b[1;33m'
base0 = b0 = Base0 = B0 = '\x1b[1;34m'
base1 = b1 = Base1 = B1 = '\x1b[1;36m'
base2 = b2 = Base2 = B2 = '\x1b[0;37m'
base3 = b3 = Base3 = B3 = '\x1b[1;37m'
yellow = y = Yellow = Y = '\x1b[0;33m'
orange = o = Orange = O = '\x1b[1;31m'
red = r = Red = R = '\x1b[0;31m'
magenta = m = Magenta = M = '\x1b[0;35m'
violet = v = Violet = V = '\x1b[1;35m'
blue = b = Blue = B = '\x1b[0;34m'
cyan = c = Cyan = C = '\x1b[0;36m'
green = g = Green = G = '\x1b[0;32m'
reset = x = Reset = X = '\x1b[0m'
screen = clear = Screen = Clear = s = S  = '\x1b[2J\x1b[H'
line = l = Line = L = '\x1b[2K\x1b[G'

// nice when picking a random color
colors = [ y, o, r, m, v, b, c, g ]

File: solarized.go
package solarized

const (
	Base03  = "\033[1;30m"
	B03     = Base03
	Base02  = "\033[0;30m"
	B02     = Base02
	Base01  = "\033[1;32m"
	B01     = Base01
	Base00  = "\033[1;33m"
	B00     = Base00
	Base0   = "\033[1;34m"
	B0      = Base0
	Base1   = "\033[1;36m"
	B1      = Base1
	Base2   = "\033[0;37m"
	B2      = Base2
	Base3   = "\033[1;37m"
	B3      = Base3
	Yellow  = "\033[0;33m"
	Y       = Yellow
	Orange  = "\033[1;31m"
	O       = Orange
	Red     = "\033[0;31m"
	R       = Red
	Magenta = "\033[0;35m"
	M       = Magenta
	Violet  = "\033[1;35m"
	V       = Violet
	Blue    = "\033[0;34m"
	B       = Blue
	Cyan    = "\033[0;36m"
	C       = Cyan
	Green   = "\033[0;32m"
	G       = Green
	Reset   = "\033[0m"
	X       = Reset
	Screen  = "\033[2J\033[H"
	S       = Screen
	Line    = "\033[2K\033[G"
	L       = Line
)

var Colors = []string{Y, O, R, M, V, B, C, G}
